### ISSUE TITLE
Windows menu: Keyboard focus is not clearily visible on the "File,Edit" etc

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51955]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Keyboard,MAS2.4.7

**Repro Steps:**
1.Navigate to "Menu Bar" at top of the screen after login.
2.Try to navigate "Menu Bar" through the keyboard key such as "TAB".
3.Observed that,when menu of "Menu Bar" is selected then the focus is not clearily visible.

**Actual:**
If the user select the menu from "Menu Bar" then keyboard focus is not clearily visible.

**Expected:**
After selecting the Menu such as "File,Edit" etc then keyboard focus should be dotted or dark.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/fixit-details.aspx?fixitid=317](url)

**User Impact:**
If the keyboard focus is very faint and if it is not in dotted format then color blind user will 
not able to see the focus.

**MAS Reference:**
MAS2.4.7- [https://microsoft.sharepoint.com/teams/msenable/mas/pages/fixit-details.aspx?fixitid=317](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_MenuBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2104842/Accessibility.A11y_VSCode_MenuBar_Keyboard.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@sbatten This should be fixed with the new menu bar on windows, correct?
If yes please close this

**Comment by sbatten** [comment]
@isidorn I don't think the new menus quite meet the criteria yet. While they are more obvious than before, I think at a minimum, the HC theme should have a bright orange border, which it does not currently

**Comment by sbatten** [comment]
resolved with #53924
